Documentation/spec/docfx_document_schema.md

```diff
@@ -0,0 +1,197 @@
+In this case, a tag can be defined to mark the property, then the processor can fetch the instances with defined tag from schema parser, and read or modify it.
+
+### 6.3 overwrite
+The value of "overwrite" MUST be a string. It specifies how DocFX uses the [overwrite files](..\tutorial\intro_overwrite_files.md) to overwrite the existing values.
+
+The following values are allowed:
+* `replace`: The value will be overwitten by the one specified in overwrite files. This is the default behavior if no overwrite keywords is specified.
+* `key`: If the instance is a string, it will be used as the key of the current object. It can be used to identify the object to overwrite in an array.
+* `merge`: If the instance is an object, it will be merged with the one in overwrite files. If there are duplicated keys, the one in overwrite files will overwrite the existing one.
+* `ignore`: Mark the current instance cannot be overwritten.
+
+> [!Note]
+> After applying the overwrite files, the model should be checked again to ensure that all the validation rules still satifies.
+
+## 7. Undefined keywords
+The schema parser can only recognize the keywords defined above. Undefined keywords is also allowed in the schema. It can provide some description for the schema, or provide more information for other parties to process.
+
+> [!note]
+> As there may be more keywords in JSON schema added to this, it's NOT RECOMMENDED to reuse the keywords already defined in JSON schema here, unless the usage is exactly same.
+
+## 8. Samples
+Here's an sample of the schema. Assume we have the following YAML file:
+```yaml
+title: Web Apps Documentation
+metadata:
+  title: Azure Web Apps Documentation - Tutorials, API Reference
+  meta.description: Learn how to use App Service Web Apps to build and host websites and web applications.
+  services: app-service
+  author: apexprodleads
+  manager: carolz
+  ms.service: app-service
+  ms.tgt_pltfrm: na
+  ms.devlang: na
+  ms.topic: landing-page
+  ms.date: 01/23/2017
+  ms.author: carolz
+sections:
+- title: 5-Minute Quickstarts
+  children:
+  - text: .NET
+    href: app-service-web-get-started-dotnet.md
+  - text: Node.js
+    href: app-service-web-get-started-nodejs.md
+  - text: PHP
+    href: app-service-web-get-started-php.md
+  - text: Java
+    href: app-service-web-get-started-java.md
+  - text: Python
+    href: app-service-web-get-started-python.md
+  - text: HTML
+    href: app-service-web-get-started-html.md
+- title: Step-by-Step Tutorials
+  children:
+  - content: "Create an application using [.NET with Azure SQL DB](app-service-web-tutorial-dotnet-sqldatabase.md) or [Node.js with MongoDB](app-service-web-tutorial-nodejs-mongodb-app.md)"
+  - content: "[Map an existing custom domain to your application](app-service-web-tutorial-custom-domain.md)"
+  - html: "[Bind an existing SSL certificate to your application](app-service-web-tutorial-custom-SSL.md)"
```

Comment:
Seems no work needed for html value. BTW, this `html` should be `content`...

---
In reply to: [122867975](https://github.com/dotnet/docfx/pull/1771#discussion_r122867975) [](ancestors = 122867975)